samples/account-linking/csharp/README.md

```diff
@@ -98,8 +100,16 @@ Please fill in the following values into the `Manifest/Manifest.json` file (call
 | Bot Id  | The "Application (client) id" from step 3. |
 | Tab domain | The domain from step 1 |
```

Comment:
If we're going add the `Base Url` `Base Url Domain` we don't need the `Tab domain` (you removed this from the manifest.json and replaced it with `Base Url`)

Reply:
Yes removed it